src/main/java/com/starbucks/controllers/RestControllers.java

```diff
@@ -20,29 +20,33 @@ public RestControllers(OrderService orderService) {
 
     @PostMapping()
     public String saveOrder(@RequestBody Order order) {
-        orderService.saveOrder ( order );
-        return order.toString ( );
+        orderService.saveOrder(order);
+        return order.toString();
     }
 
     @PutMapping("/{id}")
-    public String updateOrder(@RequestBody Order order , @PathVariable("id") UUID id) {
-
+    public String updateOrder(@RequestBody Order order, @PathVariable("id") UUID id) {
         try {
-            orderService.existOrder ( id );
+            orderService.existOrder(id);
         } catch (RuntimeException ex) {
-            throw new ResourceAccessException ( "Order does not exist " );
+            throw new ResourceAccessException("Order does not exist");
         }
-        orderService.updateOrder ( order , id );
-        return order.toString ( );
+        orderService.updateOrder(id, order);
+        return order.toString();
     }
 
     @DeleteMapping("/{id}")
     public void deleteOrder(@PathVariable("id") UUID id) {
-        orderService.deleteOrdersById ( id );
+        orderService.deleteOrdersById(id);
     }
 
-    @PatchMapping("/{id}")
-    public void patchOrder(@RequestBody Status status , @PathVariable("id") UUID id) {
```

Comment:
You should return Order here as well, just because the API user will need updated data to show a confirmation for a user and I assure you that guy won't be glad to do additional request to your API to receive data, but I can be wrong there are a lot of kinky bastards